miio/airconditioningcompanion.py

```diff
@@ -133,33 +132,25 @@ def device_type(self) -> int:
 
     @property
     def air_condition_brand(self) -> int:
-        """
-        Brand of the air conditioner.
+        """Brand of the air conditioner.
 
         Known brand ids are 0x0182, 0x0097, 0x0037, 0x0202, 0x02782, 0x0197, 0x0192.
         """
         return int(self.air_condition_model[2:4].hex(), 16)
 
     @property
     def air_condition_remote(self) -> int:
-        """
-        Known remote ids:
-
-        0x80111111, 0x80111112 (brand: 0x0182)
-        0x80222221 (brand: 0x0097)
-        0x80333331 (brand: 0x0037)
-        0x80444441 (brand: 0x0202)
-        0x80555551 (brand: 0x2782)
-        0x80777771 (brand: 0x0197)
-        0x80666661 (brand: 0x0192)
+        """Known remote ids:
 
+        0x80111111, 0x80111112 (brand: 0x0182) 0x80222221 (brand: 0x0097) 0x80333331
```

Comment:
Do you like this new format? I don't have a strong opinion on that. It looks like some docstrings arent' as readible as before now.

Reply:
Hmm, indeed, thanks for spotting! The previous one looks much better in this case, so this should definitely be fixed. 

Maybe this happens as the original got no real summary-line? Or maybe as it is not a list (in which case adding a `-` or `*` will probably fix that)?

I'll take a look into that, and adjust accordingly.

Reply:
Fixed now.